gui/doc/viselements.json

```diff
@@ -0,0 +1,1252 @@
+{
+  "controls": [
+    [
+      "text",
+      {
+        "default": "value",
```

Comment:
Why not mark the property as default: true ?

Reply:
I like that!